templates/careers/thank-you.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1nsZWk9osMxoMlSnSNIXZCFCV8-Ho4PeLXo0qD-vF_Bk/edit#{% endblock meta_copydoc %}
+
+{% block title %}Thank you{% endblock %}
+
```

Comment:
you are missing the meta description here `Thanks for taking the time to apply for our position. We appreciate your interest in Canonical. `